src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { TodoPrepared } from '../../types/TodoPrepared';
+
+import './TodoInfo.scss';
+
+type Props = {
+  todoPrepared: TodoPrepared;
+};
+
+type State = {
+  completed: boolean;
+};
+
+export class TodoInfo extends React.Component<Props, State> {
+  state = {
+    completed: false,
```

Comment:
What if on task from the server completed will be true? Will your code work correctly?